newIDE/app/src/ObjectsList/index.js

```diff
@@ -1420,6 +1433,7 @@ const ObjectsList = React.forwardRef<Props, ObjectsListInterface>(
           {
             label: i18n._(t`Duplicate`),
             click: () => duplicateObject(item),
+            accelerator: 'CTRL+D',
```

Comment:
This value does not respect the format we use, you can search `accelerator` in the repo to understand how you should write it.